src/segment_wakatime.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+)
+
+type wakatime struct {
+	props properties
+	env   environmentInfo
+
+	Hours        int
+	Minutes      int
+	MinutesTotal int
+}
+
+const (
+	// CacheKeyResponse key used when caching the response
+	WTCacheKeyResponse string = "wt_response"
+)
+
+type wtTotals struct {
```

Comment:
I would expose this as well, no? We can use inheritance to automaticaly add these after parsing the response.

Reply:
Not totally sure what you mean by inheritance, but i cound expose this, yes. Will do and we'll see if that's hat you expected ;)

Reply:
By adding that struct to the segment struct. You can then also parse the response with that embedded struct (like with git and scm).